client/my-sites/people/team-invite/index.tsx

```diff
@@ -76,7 +76,7 @@ function TeamInvite( props: Props ) {
 
 			<HeaderCake onClick={ goBack }>
 				{ translate( 'Add team members to %(sitename)s', {
-					args: { sitename: site.name },
+					args: { sitename: site.name as string },
```

Comment:
Same as above - changing the string to default to `"this site"` would make this less awkward if `site.name` is nullish for some reason.